megamek/src/megamek/client/ui/swing/lobby/LobbyErrors.java

```diff
@@ -138,5 +142,9 @@ public static void showOnlyEmptyForce(Frame owner) {
         JOptionPane.showMessageDialog(owner, FORCE_EMPTY);
     }
 
+    public static void showSBFConversion(Frame owner) {

```

Comment:
```suggestion
    public static void showSBFConversion(JFrame owner) {
```
We've had some issues in MHQ because of using Frame instead of JFrame.